test/unit/setup-wp-aliases.js

```diff
@@ -4,13 +4,15 @@ global.wp = {
 		next() {},
 		regexp: jest.fn().mockReturnValue( new RegExp() ),
 	},
+	utils: {
+		WordCounter: 'test',
+	},
 };
 
 [
 	'element',
 	'i18n',
 	'components',
-	'utils',
```

Comment:
Is there an issue with stubbing the global here with what we provide via `utils` ?

Reply:
Yes, tests were failing because for some reason the `global.utils` was empty when requiring the `@wordpress/utils` module even with the global being defined above.